projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -89,6 +89,7 @@ public static function enqueue_scripts() {
 			array(
 				'apiRoot'               => esc_url_raw( rest_url() ),
 				'apiNonce'              => wp_create_nonce( 'wp_rest' ),
+				'products'              => self::get_products(),
```

Comment:
more of a question rather than a comment, and according to what you added to the PR description:

> Introduces minimal method returning empty array for initial state of products.

What are the criteria that we are going to use to expose data to the client? Using this method, we don't need to hit the endpoint from the client but it can create a delay in the first paint.
Something to discuss in the future, surely.

Reply:
> it can create a delay in the first paint.
This is true indeed. There's a follow-up pr for the actual product date... We can discuss there. Historically at least within the Jetpack plugin a lot of data ended up being passed that way without any worry for delays. Consuming via a REST may be better